module/zfs/dbuf.c

```diff
@@ -2105,11 +2105,15 @@ dbuf_assign_arcbuf(dmu_buf_impl_t *db, arc_buf_t *buf, dmu_tx_t *tx)
 		(void) dbuf_dirty(db, tx);
 		bcopy(buf->b_data, db->db.db_data, db->db.db_size);
 		arc_buf_destroy(buf, db);
+#ifdef HAVE_UIO_ZEROCOPY
 		xuio_stat_wbuf_copied();
+#endif
```

Comment:
I should have been clearer.  Let's leave the `xuio_stat_*` interfaces in place for the moment.  We don't know if any monitoring software depends on this kstat existing so I'd prefer not to remove it.  But the `dmu_xuio_*` can certainly be disabled as you've done.